tests/Resources/ODS.robot

```diff
@@ -149,3 +149,31 @@ Get RHODS URL From OpenShift Using UI
     ...     //a[@data-test="application-launcher-item" and starts-with(@href,'https://rhods')]
     ${href}  Get Element Attribute    ${link_elements}    href
     [Return]   ${href}
+
+Verify Resources Values
+    [Documentation]    (string, string, list) -> None
+    ...    Args:
+    ...        actual(str): Field with the actual value of the resource
```

Comment:
Yes, maybe call it Verify Resources Field Values, since are the values of the field what it is verifying

Reply:
done

Reply:
Instead of having these keywords in ODS.robot, could you consider to add them to the OpenShiftLibrary instead?  I think they will be very useful for anybody using the library  (it could be done in a future PR if you agree)

About the naming and parameter order, what do you think of this?
```
OpenShift Resource Value Should Be Equal As Strings 
[Arguments]   @{resources}    ${resource_filter}    ${expected_value}

OpenShift Resource Value Should Match Regexp  
[Arguments]   @{resources}    ${resource_filter}    ${expected_pattern}

Examples:
OpenShift Resource Value Shoud Be Equal As Strings    @{grafana_pods_info}        status.phase                        Running    
OpenShift Resource Value Shoud Be Equal As Strings    @{grafana_pods_info}        status.containerStatuses[0].name    auth-proxy
OpenShift Resource Value Should Match Regexp          @{grafana_services_info}    spec.clusterIP                      ^((25[0-5]|(2[0-4]|1[0-9]|[1-9]|)[0-9])(\.(?!$)|$)){4}$
OpenShift Resource Value Should Match Regexp          @{grafana_routes_info}      spec.host                           ^(grafana-redhat-ods-monitoring.*)
```

Reply:
I like the @jgarciao 's naming proosal. Maybe i would just change from `resource_filter` to `resource_field` and add an example in [Documentation] to show how to fill the argument (like usually done in RF python code).

About moving the keywords to openshiftlibrary, I would say it is a nice idea, but I wouldn't give priority to it. Meaning that if you need these kw for this PR, let's define those here and in future development of openshiftlibrary we can move them

Reply:
About the parameters order, changing the order will cause the test to fail since RobotFramework unpacks the lists as normal variables. That means that if you add a list at the beginning of a function, the variables expected and actual will be overwritten with the value list.

Reply:
About the example in [Documentation] showing how to fill the argument, I think that I know what you mean, but to be sure, could you link a test in ods-ci to see an example of how to do it in RobotFramework?

Reply:
About the naming, @jgarciao @bdattoma, mixing your suggestions:
`OpenShift Resource Field Value Should Be Equal As Strings`  
`OpenShift Resource Field Value Should Match Regexp`
are this name ok?

Reply:
done